### ISSUE TITLE
Navigation menu type result in losing access to settings

### ISSUE THREAD

**Issue by topiasv** [sailfish-spotify/hutspot#61]
Changing the navigation menu type from the settings to anything else than the default one results in losing access to the settings page. Docked Panel with all icons for instance just loses all its icons and the only way to recover is to reset it directly using `dconf write /hutspot/navigation_menu_type 3`

**Comment by wdehoog** [comment]
Sorry for this. I'll try to fix it or disable the option.

**Comment by ksiazkowicz** [comment]
Do we still need this option?

**Comment by wdehoog** [comment]
You are right. We don't. I sort of fixed the problems but in the future we can get rid of the other options and only keep using the current default one.
